File: Versão em Python/Tarefa - Exercicio/Tarefa-Exercicio-2.py
#  Crie um algoritmo que solicite o nome
#  e o ano de nascimento de uma pessoa e imprima o nome e a sua idade.
nome = input('Digite o seu nome ')
print('--------------------------')

anoNascimento = input('Agora seu ano de nascimento ')
print('--------------------------')

x = int(anoNascimento)

print('O seu nome é ', nome, 'e você tem ', (2021 - x))

File: Versão em JavaScript/Tarefa - Exercicios/Tarefa - 1/Tarefa-Exercicio-2.js
// Crie um algoritmo que solicite o nome
// e o ano de nascimento de uma pessoa e imprima o nome e a sua idade.

const prompt = require('prompt-sync')({ sigint: true });

const nome = prompt('Digite o seu nome ');

const anoNascimento = prompt('Digite o seu ano de nascimento ');

idade = (2021 - Number(anoNascimento))

console.log(`O seu nome é ${nome} e a sua idade ${idade}`)